### PR TITLE
[skip e2e] Update mergify rule for adding centos7

### PR DESCRIPTION
**Pull request by jennyli-z**
Signed-off-by: Jenny Li <jing.li@zilliz.com>
/kind improvement

### CODE REVIEW

**Comment by jennyli-z** [comment]
/cc @cydrain 
Add centos 7 code checker into consideration for `ci-passed` label.

File: .github/mergify.yml [hunk 1]
```diff
@@ -191,6 +194,7 @@ pull_request_rules:
         - "status-success!=Code Checker AMD64 Ubuntu 18.04"
         - "status-success!=Build and test AMD64 Ubuntu 18.04" 
         - "status-success!=Code Checker MacOS 11"    
+        - "status-success=Code Checker CentOS 7"
```
Comment (LoveEachDay) [inline .github/mergify.yml:197]: Should be !=?

Comment (jennyli-z) [reply .github/mergify.yml]: modified already, please try to refresh and check again.

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**